core/src/main/scala/org/genivi/sota/core/rvi/TransferController.scala

```diff
@@ -180,7 +178,9 @@ class TransferProtocolActor(db: Database, rviClient: RviClient,
             db.run(OperationResults.persist(org.genivi.sota.core.data.OperationResult(
```

Comment:
@rdanitz Just commenting here because I think I don't understand this code and I need to extract this out so I can use it in the vehicle api, when the client sends an http request to do the same.

It seems these `db.run`s are expected to run sequentially, for example, `byId` is used after a `persist` call to UpdateRequests, but they don't necessarily run sequentially, each one creates a new `Future`, so I think this can lead to some problems??